src/crypto/OperationalKeystore.h

```diff
@@ -246,6 +244,9 @@ class OperationalKeystore
      * @brief Release an ephemeral keypair previously provided by `AllocateEphemeralKeypairForCASE()`
      */
     virtual void ReleaseEphemeralKeypair(Crypto::P256Keypair * keypair) = 0;
+
+protected:
+    ~OperationalKeystore() = default;
```

Comment:
This one NEEDS to remain virtual as it is widely deployed by ecosystems with non-default operations.

Reply:
Do you mean that ecosystems destroy this object using the base pointer?